Source/WebCore/platform/mediastream/RealtimeMediaSource.cpp

```diff
@@ -251,8 +332,29 @@ void RealtimeMediaSource::videoFrameAvailable(VideoFrame& videoFrame, VideoFrame
     updateHasStartedProducingData();
 
     Locker locker { m_videoFrameObserversLock };
-    for (auto* observer : m_videoFrameObservers)
-        observer->videoFrameAvailable(videoFrame, metadata);
+    for (auto& keyValuePair : m_videoFrameObservers) {
```

Comment:
nit, you can do `for(auto& [key, value] : ...)`

Reply:
done